src/hosted-buttons/utils.test.js

```diff
@@ -636,10 +636,20 @@ describe("buildHostedButtonOnShippingAddressChange", () => {
         status: 500,
       })
     );
-    // $FlowIssue
-    await onShippingAddressChange(data, actions);
```

Comment:
The test is titled, ""throws error when shipping-options api doesn't return 200" yet in the original test the callback does not throw. The test only verifies that the actions.reject method was called.